pkg/nsx/services/common/store.go

```diff
@@ -123,6 +124,17 @@ func TransError(err error) error {
 // InitializeResourceStore is the method to query all the various resources from nsx-t side and
 // save them to the store, we could use it to cache all the resources when process starts.
 func (service *Service) InitializeResourceStore(wg *sync.WaitGroup, fatalErrors chan error, resourceTypeValue string, store Store) {
```

Comment:
this is to search all resources under "infra" right? Could we change to InitializeInfraResourceStore?

Reply:
It's a full-text search API according to SDK, it doesn't say it's under "infra", I believe it is a global search.